css/obBase.css

```diff
@@ -3597,6 +3597,7 @@ input[type="checkbox"].fieldItem:checked + label .togLabelOff {
   position: absolute;
   top: 50%;
   right: 50%;
+  padding-top: 1px;
```

Comment:
This will affect every spinner, not just the one that wobbles, and adds a wobble to the other spinners.